tests/Basics/HashesTest.cpp

```diff
@@ -159,156 +185,156 @@ TEST_F(CHashesTest, tst_fnv64_simple) {
   std::string buffer;
 
   buffer = "";
-  EXPECT_TRUE(14695981039346656037ULL == TRI_FnvHashString(buffer.c_str()));
-  EXPECT_TRUE(14695981039346656037ULL ==
-              TRI_FnvHashPointer(buffer.c_str(), strlen(buffer.c_str())));
+  EXPECT_TRUE(14695981039346656037ULL == FnvHashString(buffer.c_str()));
   EXPECT_TRUE(14695981039346656037ULL ==
-              TRI_FnvHashBlock(TRI_FnvHashBlockInitial(), buffer.c_str(),
-                               strlen(buffer.c_str())));
+              FnvHashPointer(buffer.c_str(), strlen(buffer.c_str())));
+  EXPECT_TRUE(14695981039346656037ULL == FnvHashBlock(kFnvHashBlockInitial,
+                                                      buffer.c_str(),
+                                                      strlen(buffer.c_str())));
```

Comment:
is it possible to replace all occurences of `strlen(buffer.c_str())` with just `buffer.size()`?

Reply:
I think yes. honestly because of it in test, I just decide don't touch it